tsl/test/sql/dist_policy.sql

```diff
@@ -0,0 +1,73 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
+RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_auto_drop_chunks'
+LANGUAGE C VOLATILE STRICT;
+
+-- Cleanup from other potential tests. This avoid additional notice
+-- messages when adding data nodes.
+SET client_min_messages TO ERROR;
+DROP DATABASE IF EXISTS data_node_1;
+DROP DATABASE IF EXISTS data_node_2;
+DROP DATABASE IF EXISTS data_node_3;
+SET client_min_messages TO NOTICE;
+
+-- Add data nodes
+SELECT * FROM add_data_node('data_node_1', host => 'localhost',
+                            database => 'data_node_1');
+SELECT * FROM add_data_node('data_node_2', host => 'localhost',
+                            database => 'data_node_2');
+SELECT * FROM add_data_node('data_node_3', host => 'localhost',
+                            database => 'data_node_3');
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO PUBLIC;
+
+CREATE TABLE disttable(
+    time timestamptz NOT NULL, 
+    device int, 
+    value float
+);
+
+SELECT * FROM create_distributed_hypertable('disttable', 'time', 'device', 3);
+
+INSERT INTO disttable VALUES
+       ('2017-01-01 06:01', 1, 1.2),
+       ('2017-01-01 09:11', 3, 4.3),
+       ('2017-01-01 08:01', 1, 7.3),
+       ('2017-01-02 08:01', 2, 0.23),
+       ('2018-07-02 08:01', 87, 0.0),
+       ('2018-07-01 06:01', 13, 3.1),
+       ('2018-07-01 09:11', 90, 10303.12),
+       ('2018-07-01 08:01', 29, 64);
+
+SELECT add_retention_policy('disttable', INTERVAL '4 months', true) as drop_chunks_job_id \gset
```

Comment:
I guess a dist table cannot have a continuous aggregate (don't recall if we yet block this case)? Otherwise it would be good ro test a  policy on a continuous aggregate, even if this isn't supported yet for multinode.